internal/compiler/generator/rust.rs

```diff
@@ -76,6 +76,7 @@ fn rust_primitive_type(ty: &Type) -> Option<proc_macro2::TokenStream> {
         Type::Angle => Some(quote!(f32)),
         Type::PhysicalLength => Some(quote!(slint::private_unstable_api::re_exports::Coord)),
         Type::LogicalLength => Some(quote!(slint::private_unstable_api::re_exports::Coord)),
+        Type::Rem => Some(quote!(f32)),
```

Comment:
There is another location in this file where we need to match for this. For the comparison operator. (Remember the issue with % ?)

Reply:
Thanks - indeed.